presto-orc/src/main/java/com/facebook/presto/orc/reader/ByteSelectiveStreamReader.java

```diff
@@ -456,4 +539,60 @@ public void close()
 
         systemMemoryContext.close();
     }
+
+    private boolean useBatchMode(int positionCount, int totalPositionCount)
+    {
+        // JMH benchmark shows that when there is no null or no filter, the batch read mode was always better than the skipping mode.
+        // When there is filter and partial nulls, the batch read mode was better than the skipping mode when the input filter rate is less than 0.4
+        if (presentStream == null || filter == null || (double) (totalPositionCount - positionCount) / totalPositionCount <= 0.4) {
```

Comment:
We won't hit this branch when totalPositionCount = 0, right?
Can replace the condition with `(double)positionCount/totalPositionCount >= 0.6`

Reply:
> We won't hit this branch when totalPositionCount = 0, right?
Right. positionCount and totalPositionCount must be > 0 

> If you want you can replace the condition with `(double)positionCount/totalPositionCount >= 0.6`
@sdruzkin Thanks for the suggestion. I was trying to align with the "filter rate" convention in Presto query plans, and the filter rate in `BenchmarkSelectiveStreamReaders`. It'll be easier to match and reason about if we use the same meaning and number here.